bark/world/evaluation/BUILD

```diff
@@ -123,6 +123,35 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
```

Comment:
please put all rss related files in a folder "evaluation/rss"

Reply:
We do not always want to build the rss library (see planners)

Reply:
OK, it is solved in commit https://github.com/bark-simulator/bark/commit/c28b2cd7a5ac44335772a324ba4c1efcbebc10ea

now rss only build if `--define rss=true` is added